packages/driver/src/cy/assertions.ts

```diff
@@ -307,6 +307,8 @@ export const create = (Cypress: ICypress, cy: $Cy) => {
         // ensure the error is about existence not about
         // the downstream assertion.
         try {
+          // Ensure the command is on the same origin as the AUT
+          cy.ensureCommandCanCommunicateWithAUT(err)
```

Comment:
We talked about moving this method off `cy` and importing it directly, right? I think we should do that.

Reply:
The one we moved was 'isRunnerAbleToCommunicateWithTheAUT', it was on location. And I agree we attach too much stuff to cy/cypress when we should only be attaching public functions to them.

https://github.com/cypress-io/cypress/pull/23297/commits/eecc8302e0f8885132966bf4fbe1cbd6f2511e8f